dev_apps/first-actor/intro.md

```diff
@@ -1,4 +1,12 @@
-## wash101
+# Your First wasmcloud Actor!
 
-## Your first actor
-In this tutorial, you will deploy your first actor using the wash REPL
+Congrats on making it this far!  Now it is time to take the take what we have learn from the basics courses and build our first actor.  How about a calculator?!
+
+In this tutorial, we have provided you with 3 implementations of the same calculator app.  They are written in Rust, Golang, and AssemblyScript.  You can choose your own adventure, or you can do them all!  One of the tenants of wasmcloud is that it allows you to build on the polyglot power of Webassembly.  
+
+By the end of this tutorial you should have an understanding of:
+- How wasmcloud leverages the polyglot power of webassembly
+- Signing a wasm module with `wash`
+- Pushing a signed `wasmcloud` module to an OCI compliant registry
```

Comment:
no backticks on wasmcloud. When we say "a signed wasmCloud module" it's referring to wasmCloud as a non, not as the name of a binary file run at a command line.